core/src/test/scala/unit/kafka/integration/KafkaServerTestHarness.scala

```diff
@@ -96,4 +104,6 @@ trait KafkaServerTestHarness extends ZooKeeperTestHarness {
       alive(i) = true
     }
   }
+
+  def setClusterAcl: Unit = {}
```

Comment:
I don't quite like the maybe, so I'm proposing a different way of dealing with it, let me know if it works for you.